query-node/schemas/membership.graphql

```diff
@@ -80,6 +80,14 @@ type Membership @entity {
 
   "Content channels the member owns"
   channels: [Channel!] @derivedFrom(field: "ownerMember")
+
+  # Council & Referendum relations
+
```

Comment:
It would be helpful to have `isCouncil` flag or any other means to allow filtering members by council status. I don't remember if we could filter on the `councilMembers` field.

Reply:
`isCouncilMember` has been added to the membership.

Reply:
Thanks!

Reply:
Hey, this information is implicitly available through field `Membership::councilMembers`, is that not usable?

Reply:
We had some difficulties filtering by relation. Additionally, we would have to construct a `where` that will account for:

*   current council member
*   either is in the relationship or not (depending on what we would like to search)

In other words, the `where` clause might be complicated. Thus having this is preferable:

```graphql
query GetMemers(where:{isCouncilMember_eq:true, isVerified_eq:true}) {
  id
  handle
  #...
}
```

Reply:
There are some improvements for relation filtering coming so we can revisit this part after that. Until then we can stick with possibly redudant `isCouncilMember`.